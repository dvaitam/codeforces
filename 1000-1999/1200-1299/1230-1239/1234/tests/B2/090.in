12 9
9 5 6 8 3 10 19 17 17 18 20 18
