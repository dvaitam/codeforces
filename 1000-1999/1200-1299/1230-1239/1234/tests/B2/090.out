9
20 18 17 19 10 3 8 6 5
